packages/forms-web-app/src/routes/examination/index.js

```diff
@@ -255,7 +255,7 @@ router.post(
 	postYourInterestedPartyNumber
 );
 
-router.get(addAnotherDeadlineItem.route, getAddAnotherDeadlineItem);
+router.get(addAnotherDeadlineItem.route, unsetSubmissionItemEditMode(), getAddAnotherDeadlineItem);
```

Comment:
shouldn't have the function call  to execute on load. 

This would fire when the application loads ?

Reply:
@jonathangoulding  This is saying that however we enter this page if we are in the edit mode we inset it. Firing it in the middleware means we don't have to add extra logic on the back and continue link/button from the 'Check your answers' page before. Also if the user was to click change but at some point use the browser back button or URL to get back to the 'Add another item' page this would also remove the edit more.